mobile/src/libs/components/app/app.tsx

```diff
@@ -21,7 +22,9 @@ const App: React.FC = () => {
     <StoreProvider store={store.instance}>
       <GestureHandlerRootView style={styles.root}>
         <NavigationContainer>
-          <RootNavigation />
+          <ProtectedRoute>
+            <RootNavigation />
+          </ProtectedRoute>
```

Comment:
We don't want to make every screen protected. Screens like Sign in and Sign up should not be protected.